config.js

```diff
@@ -334,6 +313,33 @@ exports.updateFromArgv = function() {
       ["minperiodtimeunranked", "minperiodtimeblitzunranked, --minperiodtimeliveunranked and/or --minperiodtimecorrunranked"]
       ]
     deprecatedArgs.forEach(ar => testDeprecated(...ar))
+
+    let familyArray = []; 
```

Comment:
I wonder if we should

`let familyArray = familyArrayFromGeneralArg("komi");`

down after the function is specified.

Reply:
oh indeed

i forgot that we could do that in javascript

but i wonder if the familyArray autoflushes at next use now